packetbeat/protos/pgsql/parse.go

```diff
@@ -265,6 +271,53 @@ func (pgsql *Pgsql) parseErrorResponse(s *PgsqlStream, length int) (bool, bool)
 	return true, true
 }
 
+func (pgsql *Pgsql) parseParse(s *PgsqlStream, length int) (bool, bool) {
```

Comment:
maybe find a better name? parseParse sounds like the most top-level parser function. What's type `'p'` stand for?